data/ecosystems/f/flow-community.toml

```diff
@@ -13,6 +13,9 @@ github_organizations = [
   "https://github.com/punchvn",
   "https://github.com/rayvin-flow",
   "https://github.com/thirdbridge",
+  "https://github.com/decentology",
```

Comment:
let's keep this one out for now since it adds a ton of boilerplate repos

Reply:
Okay